src/pages/CartPage.vue

```diff
@@ -24,10 +26,17 @@ import CartConfirm from '/@/components/Cart/CartConfirm.vue'
 
 const cartStore = useCart()
 const router = useRouter()
-useTitle(ref('まとめて借りる'))
 
 const itemCount = computed(() => cartStore.cart.length)
 
+const title = computed(() => classification(itemCount.value))
+const classification = (number: number): string => {
+  if (number > 1) return 'まとめて借りる'
+  else if (number === 1) return '借りる'
+  else return '何も選択されていません'
+}
```

Comment:
```suggestion
const title = computed(() => {
  if (itemCount.value > 1) {
    return 'まとめて借りる'
  }
  if (itemCount.value === 1) {
    return '借りる'
  }
  return '何も選択されていません'
})
```
classificationだと何をする関数かわかりづらいので関数名を変えたいです
ただ、別にわざわざ関数を宣言する必要がないのでこんな感じにしてほしいです
他の変えた理由は以下です
- if文を一行で書くのは避けたい
- elseはなんか見た目がごちゃっとするので必要がないなら避けたい(これは本当に僕の好みで一般のルールではないです)